src/Components/SurchargeUtils.res

```diff
@@ -0,0 +1,89 @@
+type oneClickWallets = {
+  paymentMethodType: string,
+  displayName: string,
+}
+let oneClickWallets = [
+  {paymentMethodType: "apple_pay", displayName: "ApplePay"},
+  {paymentMethodType: "paypal", displayName: "Paypal"},
+  {paymentMethodType: "google_pay", displayName: "GooglePay"},
```

Comment:
@ArushKapoorJuspay @prafulkoppalkar is this an exhaustive list of one-click payment methods?

Reply:
Yes, it is @ArushKapoorJuspay only these methods can be one click checkouts right ?